zamba/models/model.py

```diff
@@ -1,19 +1,7 @@
-import pickle
 from pathlib import Path
 from shutil import rmtree
 import tempfile
 
-import pandas as pd
-
-
-try:
-    from tensorflow import keras
-except ImportError:
-    msg = "Zamba must have tensorflow installed, run either `pip install zamba[cpu]` "\
-          "or `pip install zamba[gpu]` " \
-          "depending on what is available on your system."
-    raise ImportError(msg)
```

Comment:
We need this check for the cnn ensemble in case someone does `pip install zamba` instead of `pip install zamba[cpu]` or `pip install zamba[gpu]`. Doesn't have to be in this file, but needs to happen on that codepath

Reply:
Makes sense. I'll move it back to model.py since both SampleModel and CnnEnsemble inherit from Model (and need keras)